tools/Customization/DevHome.Customization/ViewModels/FileExplorerViewModel.cs

```diff
@@ -143,6 +145,13 @@ public async Task AddFolderClick()
                 if (repoRootfolder != null && repoRootfolder.Path.Length > 0)
                 {
                     _log.Information($"Selected '{repoRootfolder.Path}' as location to register");
+                    var wrapperResult = ExtraFolderPropertiesWrapper.Register(repoRootfolder.Path, typeof(SourceControlProvider).GUID);
```

Comment:
Per the current UX design, when the user clicks Add Folder, the folder will appear as "unassigned" in UI. The user will then navigate to the drop down and selects the version control protocol of their choice if more than one is present. Therefore, the registration with File Explorer should only occur when the user makes this selection. As such, this code logic will need to be placed inside `AssignSourceControlProviderToRepository`. Thanks!